packages/react-components/src/InputAddress/index.tsx

```diff
@@ -138,7 +139,17 @@ class InputAddress extends React.PureComponent<Props, State> {
 
     // the options could be delayed, don't render without
     if (!hasOptions && !isDisabled) {
-      return null;
+      // This is nasty, but since this things is non-functional, there is not much
+      // we can do (well, wrap it, however that approach is deprecated here)
+      return (
+        <Static
+          className={className}
+          help={help}
+          label={label}
+        >
+          No accounts are available for selection.
```

Comment:
Any chance we could get a text field to paste in an account we copied from somewhere else (like a block explorer, node log, text message, etc)?

I know I'm asking a lot here, and I'm already happy with how this is. Just thought I'd share the idea.

Reply:
No. The issue is that it can be multiple -

- just accounts
- just addresses
- both of them

So there needs to be logic to check the type and then swap to the simple input component. Maybe in the future when the component is reworked, but atm piling more stuff onto something (I believe) creaky is not an area I'm willing to wander into.